test/contracts/front/challenges.js

```diff
@@ -408,6 +411,17 @@ describe('Challenges Contract', () => {
       await expect(resolveChallenge(challengeId, resolver)).to.be.eventually.fulfilled;
     });
 
+    it('Can resolve if resolver is on cooldown but challenge is older than cooldown timeout', async () => {
```

Comment:
`Can resolve ignoring resolver cooldown if challenge is older than cooldown timeout`